packages/components/src/custom-select-control-v2/custom-select.tsx

```diff
@@ -109,6 +111,8 @@ function _CustomSelect(
 		);
 
 	const contextValue = useMemo( () => ( { store, size } ), [ store, size ] );
+	const instanceId = useInstanceId( _CustomSelect );
+	const describedByID = `custom-select-described-by-${ instanceId }`;
```

Comment:
I think useInstanceId accepts a second parameter that is then used as the prefix so this coul be simplified, see for example https://github.com/WordPress/gutenberg/blob/3dfdfef3eb5cf35abf67bdce9013cc8cef5927b7/packages/components/src/box-control/index.tsx#L44